docs/modules/angular/pages/ngrx/cookbook/addingSearchCriteria/calendar.adoc

```diff
@@ -0,0 +1,109 @@
+= Add calendar input field
+
```

Comment:
Vorschlag: 
**Überschrift**
**Goal**: Add calendar input field 
Steps to take: ...

und dann die Steps vielleicht durchnummerieren, damit man mehr Guidance hat.
Step 1:
...
Step 2:
....

@SchettlerKoehler @KimFFVII  Oder meint ihr, dass dies zu viel des Guten ist?

Reply:
Finde ich gut, vor allem das durchnummerieren, da ich gerade merke, dass ich das bei autocomplete vermutlich machen muss

Reply:
Okay, cool